### ISSUE TITLE
elasticache

### ISSUE THREAD

**Issue by jdilla52** [sewenew/redis-plus-plus#245]
I'd like to connect to elasticache. However, aws strongly recommends connecting using dns. Is this possible using redis-plus-plus? Apologies as I might have totally missed something.

For example:

`
sw::redis::ConnectionOptions connection_options;
connection_options.host = "test-elasticache.2wnzk3.0002.use2.cache.amazonaws.com";
connection_options.port =  6379;
sw::redis::RedisCluster(connection_options);
`

**Comment by sewenew** [comment]
Of course, you can assign a DNS to `ConnectionOptions::host`. It will be resolved to IP address automatically.

Regards
